test/cli.js

```diff
@@ -9,7 +9,7 @@ const Code = require('code');
 const Lab = require('../');
 const Pkg = require('../package.json');
 const _Lab = require('../test_runner');
-
+const runCli = require('./run_cli');
```

Comment:
Please uppercase modules... this helps to make it easier to identify what is local and what is external